ProfileView/package.json

```diff
@@ -37,7 +37,7 @@
   "devDependencies": {
     "@babel/core": "^7.9.0",
     "@babel/preset-env": "^7.9.0",
-    "@cjaas/common-components": "1.2.1-dev.3",
+    "@cjaas/common-components": "1.2.0",
```

Comment:
the version for common-components needs to be 2.0.0, right?

Reply:
Got it